packages/bruno/PKGBUILD

```diff
@@ -0,0 +1,100 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# Initial PKGBUILD from AUR.
+# Old Maintainer: Vianney Bouchaud <aur dot vianney at bouchaud dot org>
+
+pkgname=bruno
+pkgver=1.34.2
+pkgrel=2
+pkgdesc='Opensource API Client for Exploring and Testing APIs.'
+groups=('blackarch' 'blackarch-webapp')
+arch=('x86_64')
+url="https://www.usebruno.com/"
+license=('MIT')
+_electron=electron31
+depends=(
```

Comment:
I'm not sure but there is maybe a style issue here, it should be horizontal until 80 chars.

Reply:
Fixed